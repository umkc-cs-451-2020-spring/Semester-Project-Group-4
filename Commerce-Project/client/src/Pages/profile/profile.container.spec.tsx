import React from 'react';
import { shallow } from 'enzyme';
import ProfileContainer from './profile.container';

describe('Profile Container', () => {

  it('Render the Profile Container', () => {
    const component = shallow(<ProfileContainer />);

    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});