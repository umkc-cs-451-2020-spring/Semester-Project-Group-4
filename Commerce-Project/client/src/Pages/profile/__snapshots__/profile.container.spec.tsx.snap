// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Container Render the Profile Container 1`] = `
<WithStyles(ForwardRef(Paper))
  className="makeStyles-paper-1"
  elevation={0}
>
  <WithStyles(ForwardRef(Grid))
    className="makeStyles-gridMargin-7"
    container={true}
    spacing={8}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={4}
    />
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={4}
    >
      <WithStyles(ForwardRef(FormGroup))>
        <WithStyles(ForwardRef(Typography))
          align="center"
          className="makeStyles-title-8"
          variant="h5"
        >
          Manage Profile
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(TextField))
          className="makeStyles-textFieldMargin-2"
          id="first"
          label="First Name"
          onChange={[Function]}
          value=""
        />
        <WithStyles(ForwardRef(TextField))
          className="makeStyles-textFieldMargin-2"
          id="last"
          label="Last Name"
          onChange={[Function]}
          value=""
        />
        <WithStyles(ForwardRef(TextField))
          className="makeStyles-textFieldMargin-2"
          id="phone"
          label="Phone Number"
          onChange={[Function]}
          value=""
        />
        <WithStyles(ForwardRef(TextField))
          className="makeStyles-textFieldMargin-2"
          id="password"
          label="Password"
          onChange={[Function]}
          type="password"
          value=""
        />
        <WithStyles(ForwardRef(TextField))
          className="makeStyles-textFieldMargin-2"
          id="confirm-password"
          label="Confirm Password"
          onChange={[Function]}
          type="password"
          value=""
        />
        <WithStyles(ForwardRef(Button))
          className="makeStyles-save-6"
          color="primary"
          onClick={[Function]}
          variant="contained"
        >
          Save
        </WithStyles(ForwardRef(Button))>
      </WithStyles(ForwardRef(FormGroup))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Paper))>
`;
